test/03-encryption-key.js

```diff
@@ -0,0 +1,30 @@
+const tape = require('tape')
+const crypto = require('crypto')
+const bfe = require('../')
+
+tape('03 encryption key', function (t) {
+  const key = crypto.randomBytes(32)
+
+  const values = [
+    'ssb:diffie-hellman/curve25519/' +
+      key.toString('base64').replace(/\+/g, '-').replace(/\//g, '_'),
```

Comment:
Can the utils be used here?

Reply:
It's a test - ie it doesn't need to be beautiful. I wrote the regex here first before the implementation. So I twist this more than the implementation!